scripts/react_native_pods.rb

```diff
@@ -621,20 +624,20 @@ def downloadAndConfigureHermesSource(react_native_path)
 
   hermes_tarball_url = hermes_tarball_base_url + hermes_tag
   hermes_tag_sha = %x[git ls-remote https://github.com/facebook/hermes #{hermes_tag} | cut -f 1].strip
-  hermes_tarball_path = "#{download_dir}/hermes-#{hermes_tag_sha}.tar.gz"
+  hermes_tarball_path = download_dir.join("hermes-#{hermes_tag_sha}.tar.gz")
 
   if (!File.exist?(hermes_tarball_path))
     Pod::UI.puts "[Hermes] Downloading Hermes source code..."
-    system("curl #{hermes_tarball_url} -Lo #{hermes_tarball_path}")
+    system("curl #{hermes_tarball_url} -Lo #{hermes_tarball_path.to_s}")
   end
   Pod::UI.puts "[Hermes] Extracting Hermes tarball (#{hermes_tag_sha.slice(0,6)})"
-  system("tar -zxf #{hermes_tarball_path} --strip-components=1 --directory #{hermes_dir}")
+  system("tar -zxf #{hermes_tarball_path.to_s} --strip-components=1 --directory #{hermes_dir.to_s}")
 
   # Use React Native's own scripts to build Hermes
-  system("cp #{sdks_dir}/hermes-engine/hermes-engine.podspec #{hermes_dir}/hermes-engine.podspec")
-  system("cp #{sdks_dir}/hermes-engine/utils/* #{hermes_dir}/utils/.")
+  FileUtils.cp(sdks_dir.join("hermes-engine", "hermes-engine.podspec"), hermes_dir.join("hermes-engine.podspec"))
+  FileUtils.cp_r(sdks_dir.join("hermes-engine", "utils"), hermes_dir)
```

Comment:
This section changed since the 0.69 cut. The PR for 0.69 is here: #33819

Reply:
Thanks I've tracked this 👍